pkg/labels/labels.go

```diff
@@ -8,6 +8,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"net"
+	"slices"
 	"sort"
```

Comment:
Is it worth replacing the other usage on line 169 in this file? Would a broader followup PR (or set of PRs) to replace `sort.Strings` and similar functions with `slices.Sort` be amenable?

https://github.com/cilium/cilium/blob/b5c7721274a97a37cf30291b5dc71182e3374cdb/pkg/labels/labels.go#L169

Reply:
Sure! I was focusing on `SortedList` performance and thus I applied the change just there, but it would be nice to update the entire codebase. Are you up to open a PR for that?

Reply:
Regarding the migration `golang.org/x/exp/slices` -> `slices` I've already opened https://github.com/cilium/cilium/pull/28614.
But sounds good for replacing sorting calls, thanks!